ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <react/renderer/core/propsConversions.h>
+
+#include "ActivityIndicatorProps.h"
+namespace facebook {
+namespace react {
+
+ActivityIndicatorProps::ActivityIndicatorProps(const ActivityIndicatorProps &sourceProps, const RawProps &rawProps)
+    : ViewProps(sourceProps, rawProps),
+        animating(convertRawProp(
+          rawProps,
```

Comment:
should we set the default value for these props here

Reply:
Default values are defined in header file so we can avoid here.